mozilla-release/browser/themes/shared/tabs.inc.css

```diff
@@ -617,7 +617,6 @@
 /* Tab separators */
 
 .titlebar-spacer[type="pre-tabs"] {
-  border-inline-end: 1px solid var(--lwt-background-tab-separator-color, currentColor);
```

Comment:
lets comment it instead? (just to avoid conflicts :P )

Reply:
@tsl143 yes, sure. Fixed.